bin/list-all

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
 releases_path=https://api.github.com/repos/elixir-lang/elixir/releases
-# Fetch all tag names, and get only second column. Then remove all unnecesary characters.
-versions=$(curl -s $releases_path | grep "tag_name" | grep -oE '[0-9](\.[0-9]){2}' | tr '\n' ' ')
-echo $versions
+
+releases=$(curl -qs $releases_path | tr ',' '\n' | grep '"tag_name":')
+releases=${releases//\"tag_name\":/}
+releases=${releases//[\",]/}
+releases=${releases//v/}
+
+echo -e $releases
```

Comment:
Actually, ignore my previous comment. This works fine. Sorting is already taken care of.